lib/manager/actions/labels.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import { secureFetch } from '../../common/actions'
+import type { dispatchFn, getStateFn } from '../../types/reducers'
+import type { Label } from '../../types'
+
+import { fetchProject } from './projects'
+import { fetchProjectFeeds } from './feeds'
+
+// Public action used by component or other actions
+
+const LABEL_URL = '/api/manager/secure/label'
+
+/**
+ * Create new label from provided properties.
+ */
+export function createLabel (label: Label) {
+  return function (dispatch: dispatchFn, getState: getStateFn) {
+    return dispatch(secureFetch(LABEL_URL, 'post', label))
+      .then(async (res) => {
+        const createdLabel = await res.json()
+        dispatch(fetchProject(createdLabel.projectId))
+        dispatch(fetchProjectFeeds(createdLabel.projectId))
```

Comment:
Each of these CRUD actions may need to be structured differently so that deployments are also fetched. See `updateProject`:
https://github.com/ibi-group/datatools-ui/blob/4df5f69a195d0c1828a391e6dbba4befa82c5b24/lib/manager/actions/projects.js#L150-L151

Reply:
Thank you! Added.